.github/workflows/windows-periodic-trigger.yml

```diff
@@ -0,0 +1,25 @@
+# Workflow intended to periodically run the Windows Integration test workflow.
+
+name: Windows Periodic Tests
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 1 * * *"
+
+jobs:
+
+  triggerWinIntegration:
+    if: github.repository == 'containerd/containerd'
```

Comment:
Does users need to change this to run the workflow on fork?

Reply:
Hello @jonyhy96.

The above check makes it so that the automatic scheduling of the Windows Periodic workflow will only kick in on the main `containerd` repo. (note that the check was moved within a new `windows-periodic-trigger.yml` workflow file, whose sole purpose is to trigger the old `windows-periodic.yml` workflow based on the schedule)

Contributors with forks will still be able to manually trigger the original `windows-periodic.yml` workflow on their forks at any time.